controllers/homeController.js

```diff
@@ -19,15 +19,13 @@ router.post('/', function(req, res) {
             var summonerStats = results[0]
             var summonerStats2 = results[1]
             var commonChampions = findCommonChampions(summonerStats, summonerStats2)
-            console.log(commonChampions);
 
             var compareChampionsResult = compareCommonChampions(commonChampions);
             displayComparisonResults(summonerName, summonerName2, compareChampionsResult);
-            res.render('results', { winner: compareChampionsResult.compare, resultList:compareChampionsResult.championCompareList });
-
+            res.render('results', { summonerA: summonerName, summonerB: summonerName2, winner: compareChampionsResult.compare, resultList:compareChampionsResult.championCompareList });
```

Comment:
To me displayComparisonResults is just there for console logging at the moment and will go away later on, weare starting display results in the view now anyways.